File: lang_py/Performance.py
from Gpu import Gpu

class Performance:

    def __init__(self, kodePerformance, gpuList):
        self.kodePerformance = kodePerformance
        self.gpuList = gpuList

    def setKodePerformance(self, kodePerformance):
        self.kodePerformance = kodePerformance

    def getKodePerformance(self):
        return self.kodePerformance
    
    def setGpuList(self, gpuList):
        self.gpuList = gpuList

    def getGpuList(self):
        return self.gpuList
    
    def addGpu(self, gpu):
        self.gpuList.append(gpu)

File: lang_java/Performance.java
import java.util.List;

public interface Performance {

    // Deklarasi metode getter dan setter untuk kodePerformance
    String getKodePerformance();
    void setKodePerformance(String kodePerformance);
    
    // Deklarasi metode getter dan setter untuk gpuList
    List<Gpu> getGpuList();
    void setGpuList(List<Gpu> gpuList);
    
    // Deklarasi metode untuk menambahkan Gpu ke dalam list
    void addGpu(Gpu gpu);
}
